### DISCUSSION TITLE
bug: AddSqlServerDbContext not found

### DISCUSSION THREAD

**Discussion by Hona** [microsoft/aspire#3515] [Q&A]
I've got

```cs
var appDb = builder
    .AddSqlServer("db")
    .AddDatabase("appdb");

builder.AddProject<Projects.VerticalSliceArchitectureTemplate>("webapi")
    .WithReference(appDb);

builder.Build().Run();
```

with the following (on .NET 8)

```xml
        <PackageReference Include="Aspire.Microsoft.EntityFrameworkCore.SqlServer" Version="8.0.0-preview.4.24156.9" />
```

And the component-consuming project is trying to register the EF Core piece

```cs
        services.AddSqlServerDbContext<AppDbContext>("appdb");
```

Which doesn't work.

![image](https://github.com/dotnet/aspire/assets/10430890/f9661920-2c51-4a44-b5fd-de4da1efad95)

I tried and there's no import I'm missing (that Rider could detect, at least)

**Comment by mitchdenny** [comment]
Can you post a link to a repro repository so we can take a look?

**Comment by mitchdenny** [comment]
Also just want to check, did you add the package reference to the right project?
